hack/test-unit.sh

```diff
@@ -63,12 +63,28 @@ function test::unit() {
   fi
 }
 
+function test::cli_manifest_gen() {
```

Comment:
I think it's not really fits to the `test-unit` target. It's purely about executing command without cleanup.

It would be great to rewrite that in Go as a simple smoke test (comparing the output files to some golden files, similarly what we have for rendering). maybe as a part of #417 if we have time. For now I would delete that, honestly. What do you think?